scripts/updateCalendar.py

```diff
@@ -52,7 +50,6 @@
 DEFAULT_END_TIME = "16:30" # will be overridden by endTime in _posts
 REQUIRED_FIELDS = [ 'title', 'location', 'text', 'link', 'date' ]
 
-
```

Comment:
Two blank lines here should be PEP8 compliant (though I don't know if the rest of the file is or isn't.)